test/e2e/antreapolicy_test.go

```diff
@@ -144,6 +144,21 @@ func cleanupDefaultDenyNPs(k8s *KubernetesUtils, namespaces []string) error {
 	return nil
 }
 
+func testInvalidACNPNoPriority(t *testing.T) {
+	invalidNpErr := errors.New("invalid Antrea ClusterNetworkPolicy accepted")
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-no-priority").SetAppliedToGroup(map[string]string{"pod": "a"}, nil, nil, nil)
+	np := metav1.Object{builder.Get()}
```

Comment:
You do not need to cast it to Object and back to ACNP. An ACNP Spec without priority is valid from the compiler point of view, we just need to validate this against webhook.

Reply:
ya realized it and already changed that part